### ISSUE TITLE
Please Release Sinatra 2.0.8

### ISSUE THREAD

**Issue by jeremyevans** [sinatra/sinatra#1583]
Currently, Sinatra is blocking work on Ruby 3 keyword argument separation (https://github.com/ruby/ruby/pull/2794).  This is because Sinatra is used by bundler's tests, and bundler's tests fails with sinatra 2.0.7 when keyword arguments are separated.  I think the Sinatra master branch should fix the bundler test failure, since the Sinatra keyword separation issue was fixed in https://github.com/sinatra/sinatra/commit/d423f565ca9f461f3c2f1fe8bb08f708fef5d4e5.

**Comment by namusyaka** [comment]
Before releasing 2.0.8, please make sure that the failing bundler's test can be fixed by using Sinatra's master branch.
I don't want to hear a kind of this request again and again.

**Comment by jeremyevans** [comment]
I understand, and I agree it is best to check this first. The 133 bundler test failures in Ruby's CI all appear to be due to:

```
  /home/runner/work/ruby/ruby/src/tmp/1/gems/base/gems/mustermann-1.1.0/lib/mustermann.rb:74:in `new': Hash can't be coerced into Mustermann::Pattern (TypeError)
         	from /home/runner/work/ruby/ruby/src/tmp/1/gems/base/gems/mustermann-1.1.0/lib/mustermann.rb:71:in `block in new'
         	from /home/runner/work/ruby/ruby/src/tmp/1/gems/base/gems/mustermann-1.1.0/lib/mustermann.rb:71:in `map'
         	from /home/runner/work/ruby/ruby/src/tmp/1/gems/base/gems/mustermann-1.1.0/lib/mustermann.rb:71:in `new'
         	from /home/runner/work/ruby/ruby/src/tmp/1/gems/base/gems/sinatra-2.0.7/lib/sinatra/base.rb:1641:in `compile'
         	from /home/runner/work/ruby/ruby/src/tmp/1/gems/base/gems/sinatra-2.0.7/lib/sinatra/base.rb:1629:in `compile!'
         	from /home/runner/work/ruby/ruby/src/tmp/1/gems/base/gems/sinatra-2.0.7/lib/sinatra/base.rb:1271:in `error'
         	from /home/runner/work/ruby/ruby/src/tmp/1/gems/base/gems/sinatra-2.0.7/lib/sinatra/base.rb:1831:in `<class:Base>'
```

I updated my bundler checkout to master and ran `rspec spec/install/gemfile/gemspec_spec.rb`.  This fails with 3 errors similar to the above.  Unfortunately, even if you uninstall sinatra 2.0.7, build a gem from the sinatra master  branch and install it locally as sinatra 2.0.8, bundler still tries use sinatra 2.0.7 installed from rubygems.org.  I was able to hack around it by manually overwriting the files in `tmp/1/gems/base/gems/sinatra-2.0.7` in the bundler checkout to be files from sinatra's master branch.  After that change, there are no failures.  So I believe that bundler's tests will be fixed with sinatra's master branch released as 2.0.8.

**Comment by namusyaka** [comment]
I see, thanks for the confirmation.
Will cut a new release soon.
